Assets/MRTK/Tools/BuildWindow/BuildDeployWindow.cs

```diff
@@ -837,26 +788,6 @@ private void RenderConnectionButtons()
             }
         }
 
-        private void RenderLocalConnection()
-        {
-            using (var c = new EditorGUI.ChangeCheckScope())
-            {
-                string target = IsHoloLensConnectedUsb ? HOLOLENS_USB : DeviceInfo.LocalMachine;
```

Comment:
I've been thinking through this, and `Local` is also used to deploy things like x64 to your local PC for VR. I'm not sure removing it wholesale like this is the best move. I might be missing it, but I don't see a new codepath restoring this feature.